Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -1813,6 +1813,191 @@ def test_module(client: Client, *_) -> Tuple[str, Dict[Any, Any], Dict[Any, Any]
+    entries = []
+
+    sys_param_limit = args.get('limit', client.sys_param_limit)
+    sys_param_offset = args.get('offset', client.sys_param_offset)
+
+    sys_param_query = f'element_id={ticket_id}^element=comments^ORelement=work_notes'
+
+    comments_result = client.query('sys_journal_field', sys_param_limit, sys_param_offset, sys_param_query)
+    demisto.debug(f'Comments result is {comments_result}')
+
+    if not comments_result or 'result' not in comments_result:
+        demisto.debug(f'Pull result is {ticket}')
+        return [ticket]
+
+    for note in comments_result.get('result'):
+        entry_time = arg_to_timestamp(
+            arg=note.get('sys_created_on'),
+            arg_name='sys_created_on',
+            required=False
+        )
+        demisto.debug(f'entry_time is {entry_time}')
+
+        if last_update > entry_time:
+            continue
+
+        entries.append({
+            'Type': 1,
+            'Category': 'chat',
+            'ContentsFormat': 'text',
+            'Contents': note.get('value'),
+            'Note': True
+        })
+
+    original_ticket = result['result']
+
+    if original_ticket.get('resolved_by') != '':
+        demisto.info(f'ticket is closed: {original_ticket}')
+        entries.append({
+            'Type': EntryType.NOTE,
+            'Contents': {
+                'dbotIncidentClose': True,
+                'closeReason': f'From ServiceNow: {original_ticket.get("close_notes")}'
+            },
+            'ContentsFormat': EntryFormat.JSON
+        })
+
+    demisto.debug(f'Pull result is {ticket} + {entries}')
+
+    return [ticket] + entries
+
+
+def update_remote_system_command(client: Client, args: Dict[str, Any]) -> str:
+    """update-remote-system command: pushes local changes to the remote system
+
+    :type client: ``Client``
+    :param Client: XSOAR client to use
+
+    :type args: ``Dict[str, Any]``
+    :param args:
+        all command arguments, usually passed from ``demisto.args()``.
+        ``args['data']`` the data to send to the remote system
+        ``args['entries']`` the entries to send to the remote system
+        ``args['incidentChanged']`` boolean telling us if the local incident indeed changed or not
+        ``args['remoteId']`` the remote incident id
+
+    :return:
+        ``str`` containing the remote incident id - really important if the incident is newly created remotely
+
+    :rtype: ``str``
+    """
+    data = args.get('data')
+    entries = args.get('entries')
+    incident_changed = args.get('incidentChanged')
+    ticket_id = args.get('remoteId')
+    org_status = args.get('status')
+
+    ticket_type = 'incident'
+    if incident_changed:
+        fields = get_ticket_fields(data, ticket_type=ticket_type)
+        fields.update(additional_fields)
+
+        demisto.debug(f'Sending update request to server {ticket_type}, {ticket_id}, {fields}\n')
+        result = client.update(ticket_type, ticket_id, fields, {})
```

Comment:
```suggestion
        result = client.update(ticket_type, ticket_id, fields)
```

Reply:
The function `def update` takes 4 arguments, that's why I'm passing an empty dict